bluesky/traffic/route.py

```diff
@@ -886,152 +847,145 @@ def newcalcfp(self):
         """Do flight plan calculations"""
 
         # Remove old top of descents and old top of climbs
-        while self.wpname.count("T/D")>0:
+        while self.wpname.count("T/D") > 0:
             self.delwpt("T/D")
 
-        while self.wpname.count("T/C")>0:
+        while self.wpname.count("T/C") > 0:
             self.delwpt("T/C")
 
         # Remove old actual position waypoints
-        while self.wpname.count("A/C")>0:
+        while self.wpname.count("A/C") > 0:
             self.delwpt("A/C")
 
         # Insert actual position as A/C waypoint
         idx = self.iactwp
-        self.insertcalcwp(self,idx,"A/C")
-        self.wplat[idx] = bs.traf.lat[self.iac] # deg
-        self.wplon[idx] = bs.traf.lon[self.iac] # deg
-        self.wpalt[idx] = bs.traf.alt[self.iac] # m
-        self.wpspd[idx] = bs.traf.tas[self.iac] # m/s
+        self.insertcalcwp(idx, "A/C")
+        self.wplat[idx] = bs.traf.lat[self.iac]  # deg
+        self.wplon[idx] = bs.traf.lon[self.iac]  # deg
+        self.wpalt[idx] = bs.traf.alt[self.iac]  # m
+        self.wpspd[idx] = bs.traf.tas[self.iac]  # m/s
 
         # Calculate distance to last waypoint in route
-        dist2go = []
-        nwp = len(wpname)
+        nwp = len(self.wpname)
         dist2go = [0.0]
-        for i in range(nwp-2,-1,-1):
-            qdr,dist = geo.qdrdist(self.wplat[i],self.wplon[i],    \
-                        self.wplat[i+1],self.wplon[i+1])
-            dist2go = [dist2go[i+1]+dist]+dist2go
+        for i in range(nwp - 2, -1, -1):
+            qdr, dist = geo.qdrdist(self.wplat[i], self.wplon[i],
+                                    self.wplat[i + 1], self.wplon[i + 1])
+            dist2go = [dist2go[0] + dist] + dist2go
 
         # Make VNAV WP list with only waypoints with altitude constraints
         # This list we will use to find where to insert t/c and t/d
         alt = []
-        x   = []
+        x = []
         name = []
         for i in range(nwp):
-            if self.wpalt[i]>-1.:
+            if self.wpalt[i] > -1.:
                 alt.append(self.wpalt[i])
                 x.append(dist2go[i])
-                name.append(self.wpname[i]+" ")    # space for check first 2 chars later
+                name.append(self.wpname[i] + " ")
+                # space for check first 2 chars later
 
         # Find where to insert cruise segment (if any)
 
         # Find longest segment without altitude constraints
 
+        desslope = clbslope = 1.

```

Comment:
code analyzer complains about possible non-initialization of `desslope` and `clbslope`